Frontend/src/pages/Dashboard.jsx

```
@@ -26,12 +27,28 @@ const Dashboard = () => {
       </div>
 
-      {/* Suggested Users Section */}
-      <div className="p-6">
-        <SuggestedUsers />
+      {/* Main Dashboard Content */}
+      <div className="flex flex-col items-center justify-center flex-grow">
+        <h1 className="text-2xl font-bold mb-6">Welcome to Your Dashboard</h1>
+        <button 
+          onClick={() => navigate("/dashboard/suggested-users")} 
+          className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-3 px-6 rounded-lg shadow-lg transition duration-300">
+          👥 Pairs
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)